src/components/LoginModal.tsx

```diff
@@ -0,0 +1,214 @@
+import {
```

Comment:
Prototype:
![image](https://github.com/user-attachments/assets/e7f55424-f791-4e12-993e-ae93ac94f43c)

Actual:
![image](https://github.com/user-attachments/assets/1628aa55-be03-4e00-b247-06a6bcae135a)

Suggestions:
- More padding to the outer most border
- If possible, decrease the gap between the components
- Center the bottom text